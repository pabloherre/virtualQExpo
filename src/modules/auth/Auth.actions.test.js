import { logout, setUser } from './Auth.actions';

describe('Auth actions', () => {
  it('should create an action to logout', () => {
    expect(logout()).toMatchSnapshot();
  });
});
