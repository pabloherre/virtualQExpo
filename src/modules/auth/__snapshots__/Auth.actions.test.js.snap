// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth actions should create an action to logout 1`] = `
Object {
  "type": "LOGOUT",
}
`;
